code/game/objects/items/melee/baton.dm

```diff
@@ -731,3 +741,32 @@
 	if(!. || target.move_resist >= MOVE_FORCE_OVERPOWERING)
 		return
 	do_teleport(target, get_turf(target), 15, channel = TELEPORT_CHANNEL_BLUESPACE)
+
+/obj/item/melee/baton/security/cattleprod/telecrystalprod
+	name = "snatcherprod"
+	desc = "A prod with a telecrystal on the end. It sparks with a desire for theft and subversion."
+	w_class = WEIGHT_CLASS_NORMAL
+	icon_state = "telecrystalprod"
+	inhand_icon_state = "telecrystalprod"
+	slot_flags = null
```

Comment:
Any reason why we want the flags and w_class to be different from the regular prod?

Reply:
Consistency with teleprod rather than cattleprod. Look, don't ask me, it's just what it is.

Reply:
👍